sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseIndexDto.java

```diff
@@ -391,10 +393,15 @@ public CaseReferenceDto toReference() {
 	}
 
 	@Override
-	public String toString() {
+	public String getCaption() {
 		return CaseReferenceDto.buildCaption(getUuid(), getPersonFirstName(), getPersonLastName());
 	}
 
+	@Override
+	public String toString() {
+		return CaseDataDto.I18N_PREFIX + StringUtils.SPACE + uuid;
```

Comment:
Example where default implementation of `toString()` in `PseudonymizableIndexDto` would be helpful